melpa-packages.sh

```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/env nix-shell
-#!nix-shell --pure -i bash -A env
+#!nix-shell -i bash -A env
 
 # usage: ./melpa-packages.sh --melpa PATH_TO_MELPA_CLONE
 
+## env var for curl
+export SSL_CERT_FILE=/etc/ssl/certs/ca-certificates.crt
```

Comment:
I guess we could, but at least on my shell, SSL_CERT_FILE isn't defined. I don't think it needs to be set again in the fetcher, though.